### DISCUSSION TITLE
Question about the “threshold” of bug reports

### DISCUSSION THREAD

**Discussion by pgundlach** [pdf-association/pdf-issues#422] [General]
I am uncertain how low the threshold is for reporting bugs to the ISO PDF standard. There are many small formatting issues, but I doubt that it is worthwhile reporting/fixing them, unless you explicitly ask for these too.

For example in clause 12.5.5 right before clause 12.5.6 there is an italic word where the first letter is from an upright font.

<img width="435" alt="form" src="https://github.com/pdf-association/pdf-issues/assets/209434/3254bd47-2aa7-4874-8fb6-41acb8f2b53e">

I am sure that nobody ever bothers about that.

But can you state at which point it is worth reporting bugs? I assume that everything that is incorrect should be reported. But enhancements in formatting that help understanding aspects of PDF could be improved?

Thanks, Patrick

**Comment by petervwyatt** [comment]
We appreciate all reports, no matter how minor.

To make things easier, please feel free to bundle minor formatting issues (such as the example you give above) into a single generic issue. We may not publish them in with the full errata at https://pdf-issues.pdfa.org/ so that it can focus on errors, misunderstandings, and ambiguities (the exception would be if the formatting causes confusion, such as a key name being italic instead of bold). 

But all issues, including these smaller ones, help us get the next PDF specification to be even better quality (whenever that may be).
